scripts/test_lib.sh

```diff
@@ -108,7 +108,7 @@ function relativePath {
 # go_srcs_in_module [package]
 # returns list of all not-generated go sources in the current (dir) module.
 function go_srcs_in_module {
-  go fmt -n "$1"  | grep -Eo "([^ ]*)$" | grep -vE "(\\_test.go|\\.pb\\.go|\\.pb\\.gw.go)"
```

Comment:
Looking at: https://github-wiki-see.page/m/koalaman/shellcheck/wiki/SC2016

We need to change quotes to avoid warning: 
```
go list -f "{{with \$c:=.}}{{range \$f:=\$c.GoFiles }}{{\$c.Dir}}/{{\$f}}{{\"\n\"}}{{end}}{{end}}" ./...
```

Reply:
thanks @ptabor , updated PR